### PR TITLE
DOC: Adds documentation, including CI tests for docs

### PR DESCRIPTION
**Pull request by jklenzing**
Addresses #31, #32, #33,  #62; Replaces #37

- Adds basic documentation according to pysat standards
- Integrates with readthedocs -> https://pysatnasa.readthedocs.io/en/doc-updates/index.html
- Adds testing in Github Actions for doc build and links
- Adds badges for readthedocs and pypi
- Updates zenodo standards

NOTE: This is based off an earlier commit in #37.  That branch was broken while trying to improve the autodoc implementation.

### CODE REVIEW

**Comment by jklenzing** [comment]
Running into an issue with linkcheck.  The notes on temperature errors links to a web page that uses http (not https).  linkcheck updates this use https and then fails because you can't find the website that way.

http://saber.gats-inc.com/temp_errors.php

File: .zenodo.json [hunk 1]
```diff
@@ -6,26 +6,27 @@
...
       },
       {
-        "affiliation": "University of Texas at Dallas",
+        "affiliation": "The University of Texas at Dallas",
         "name": "Russell Stoneback",
         "orcid": "0000-0001-7216-4336"
       },
+      {
+        "affiliation": "Catholic University of America, Goddard Space Flight Center",
+        "name": "Jonathon Smith",
```
Comment (aburrell) [inline .zenodo.json:15]: Recommend reversing name order here and elsewhere: "Smith, Jonathon"

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+#html_static_path = ['.static']
+
+
+# -- Options for HTMLHelp output ------------------------------------------
+
+# Output file base name for HTML help builder.
+htmlhelp_basename = 'pysatNASAdoc'
```
Comment (aburrell) [inline docs/conf.py:116]:
```suggestion
htmlhelp_basename = '{:s}doc'.format(project)
```

Comment (aburrell) [reply docs/conf.py:116]: And a few places below.

**Comment by jklenzing** [comment]
Found out I already documented the autodoc issues in #60.

**Comment by jklenzing** [comment]
OK, there was a second http only link.  Both are working now.

File: README.md [hunk 3]
```diff
@@ -21,14 +50,7 @@ python setup.py install
...
 ------------------
-pysatNASA is currently in an initial development phase.  Much of the API is being built off of the upcoming pysat 3.0.0 software in order to streamline the usage and test coverage.  This version of pysat is planned for release later this year.  Currently, you can access the develop version of this through github:
-```
-git clone https://github.com/pysat/pysat.git
-cd pysat
-git checkout develop-3
-python setup.py install
-```
-It should be noted that this is a working branch and is subject to change.
+pysatNASA is currently in an initial development phase and requires pysat 3.0.0.
```
Comment (aburrell) [inline README.md:53]: Header underline length suspect

**Comment by jklenzing** [comment]
Sorry, I thought I applied the suggestion.  Not sure why that didn't show up.

**Pull request merged**